test/libsolidity/syntaxTests/inheritance/override/override_ambiguous.sol

```diff
@@ -0,0 +1,14 @@
+contract A {
+	int public testvar;
+	function foo() internal returns (uint256);
+}
+contract B {
+	function foo() internal returns (uint256);
+	function test() internal returns (uint256);
+}
+contract X is A, B {
+	int public override testvar;
+	function test() internal override returns (uint256);
+}
+// ----
+// TypeError: (184-290): Derived contract must override function foo. Function with the same name and parameter types defined in two or more base classes.
```

Comment:
```suggestion
// TypeError: (184-290): Derived contract must override function "foo". Function with the same name and parameter types defined in two or more base classes.
```

Reply:
hmm other error messages didn't use quotes around the function name...

Reply:
They do, e.g. `syntaxTests/array/no_array_pop.sol`
```
contract C {
    uint data;
    function test() public {
      data.pop();
    }
}
// ----
// TypeError: (63-71): Member "pop" not found or not visible after argument-dependent lookup in uint256.
```